napari/utils/colormaps/colormaps.py

```diff
@@ -45,6 +46,38 @@ def _all_rgb():
 LABRNG = LABMAX - LABMIN
 
 
+def convert_vispy_colormap(colormap, name=None):
+    """Convert a vispy colormap object to a napari colormap.
+
+    Parameters
+    ----------
+    colormap : vispy.color.Colormap
+        Vispy colormap object that should be converted.
+    name : str
+        Name of colormap, optional.
+
+    Returns
+    -------
+    napari.utils.Colormap
+    """
+    if hasattr(colormap, '_controls'):
```

Comment:
I think it would be reasonable to just check `isinstance(colormap, VispyColormap)` and throw a `TypeError` otherwise, rather than checking hasattr.  This function is, afterall, dedicated to vispy objects.